### DISCUSSION TITLE
Lien MAPS non fonctionnel

### DISCUSSION THREAD

**Discussion by danidass91** [opengisch/QField#4498] [Q&A]
J'aimerai savoir pourquoi le lien MAPS n'est plus fonctionnel et est grisé alors que avant il fonctionnait correctement.
Merci de votre aide
![IMG_20230731_103556](https://github.com/opengisch/QField/assets/141016477/3ddd26c6-cdc8-440f-913b-16b0b2ea25a4)

**Comment by nirvn** [comment]
@danidass91 , can you share the configuration of your Maps field (a screenshot of the editor widget configuration panel in QGIS will do).

**Reply by danidass91** [reply]
![IMG-20230731-WA0009](https://github.com/opengisch/QField/assets/141016477/c7f7a3e7-726d-402d-b00e-dbf4a038ed6e)

**Comment by danidass91** [comment]
![IMG-20230731-WA0009](https://github.com/opengisch/QField/assets/141016477/79e628fd-9b9a-4609-8aff-dbd312666490)

**Reply by nirvn** [reply]
@danidass91 , :) I was referring to something like this:

![image](https://github.com/opengisch/QField/assets/1728657/0ee45092-34fb-45ba-963a-8c31496049dc)

**Reply by danidass91** [reply]
![IMG-20230731-WA0011](https://github.com/opengisch/QField/assets/141016477/4542c16a-26b1-4455-a66e-2289c7db3443)

Le voici

**Reply by nirvn** [reply]
@danidass91 , I can't replicate here:

![image](https://github.com/opengisch/QField/assets/1728657/63cf42e6-f910-4fb2-85a5-7f16f7e046f7)

Which version of QField are you using?

**Reply by danidass91** [reply]
![Screenshot_2023-08-01-10-39-03-01_2e601d870d31f598a7096b1e53961eb8](https://github.com/opengisch/QField/assets/141016477/3b2e6c5b-efa4-40a8-89b8-2d0640d82331)

J'utilise celle ci, la capture d'avant a été pris via Qgis sur mon PC et non Qfield car je ne sais pas comment sur Qfield paramétré pour avoir des lien URL
